Sources/SwiftParser/Patterns.swift

```diff
@@ -379,7 +379,7 @@ extension Parser.Lookahead {
         && !self.at(.keyword(.consuming))
         && !(experimentalFeatures.contains(.transferringArgsAndResults) && self.at(.keyword(.transferring)))
         && !(experimentalFeatures.contains(.sendingArgsAndResults) && self.at(.keyword(.sending)))
-        && !(experimentalFeatures.contains(.nonescapableTypes) && self.at(.keyword(._resultDependsOn)))
+        && !(experimentalFeatures.contains(.nonescapableTypes))
```

Comment:
I think you need to remove the check for experimental features as well here

Reply:
Done. Thanks for catching that